image_editor/core/image_processing/Image_Processing.py

```diff
@@ -37,13 +40,14 @@ def crop_image(self, **kwargs) -> str:
         new_img = self.img.copy()
         height, width = self.img.shape[:2]
         file_extension = None
-        if kwargs.get('aspect_ratio', 'custom') == 'custom':
+        aspect_ratio = kwargs.get('aspect_ratio', ie.CROP_CUSTOM)
+        if aspect_ratio == ie.CROP_CUSTOM:
             left, top, right, bottom = kwargs.get('dimension', (0, 0, 0, 0))
             right = width - right
             bottom = height - bottom
             new_img = new_img[top:bottom, left:right, :]
-        elif kwargs.get('aspect_ratio', '1:1') == '1:1':
-            shift = kwargs.get('dimension', 0)
+        elif aspect_ratio == ie.CROP_1_1:
+            shift = kwargs.get('shift', 0)
```

Comment:
`shift` can it be made a constant, and all the expected KWARGS be made as a constant?
like `aspect_ratio, amount , direction ...`